src/modules/etc.ts

```diff
@@ -39,4 +40,15 @@ export class EtcModule extends Module {
 		await msg.react('❌');
 		await msg.react('🤷');
 	}
+
+	@listener({ event: 'messageReactionAdd' })
+	async onReact(reaction: MessageReaction, member: GuildMember) {
+		if (
+			reaction.emoji.name === '❌' &&
+			ownsBotMessage(reaction.message, member.id)
+		) {
+			clearMessageOwnership(reaction.message);
+			await reaction.message.delete();
```

Comment:
Since you seem to be going for most of the abstraction in the `send` file, maybe just call `Message#delete` inside `clearMessageOwnership`.

Reply:
That mixes up responsibilities... deleting a message isn't necessarily the same as clearing ownership on that message. Today we only clear ownership if deleting a message, but that might not always be true. It might make sense someday to have a message be created by Alice, and then modified by Bob, at which point Bob would be a more reasonable owner.

Moving the ownership check + delete + clear to an `attemptMessageDelete` / `deleteOwnedMessage` function would be better if you'd still like to rework the react logic.

Reply:
Ah okay, sorry for the redundant comment.